integration-cli/docker_cli_attach_test.go

```diff
@@ -96,9 +94,8 @@ func (s *DockerSuite) TestAttachTtyWithoutStdin(c *check.C) {
 
 	defer func() {
 		cmd := exec.Command(dockerBinary, "kill", id)
-		if out, _, err := runCommandWithOutput(cmd); err != nil {
-			c.Fatalf("failed to kill container: %v (%v)", out, err)
-		}
+		out, _, err := runCommandWithOutput(cmd)
```

Comment:
I think this deferrer is not needed. Because the containers will be removed in TeardownTest. WDYT?